src/DetailsView/components/reflow-assessment-view.tsx

```diff
@@ -28,7 +28,7 @@ export interface ReflowAssessmentViewProps {
     currentTarget: Tab;
     prevTarget: PersistedTabInfo;
     assessmentTestResult: AssessmentTestResult;
-    isEnabled: boolean;
+    selectedRequirementIsEnabled: boolean;
```

Comment:
Out of curiosity, why was `isEnabled`/`selectedRequirementIsEnabled` added to this component if it's value is not being used by the component at all? This should probably be removed and added when the view actually uses the value.

Reply:
it's used by the updateHandler, and will most likely be needed by RequirementView when we add that here. I'll take it out for now though (or make it clearer that it goes to updateHandler).

Reply:
I got that it's used by the updateHandler! If it's not used in the component currently though, it might make more sense to remove it from the props now and add it back once the ReflowAssessmentView is using it.